lib/workers/repository/update/pr/changelog/source-github.ts

```diff
@@ -121,7 +121,7 @@ export async function getChangeLogJSON(
 
   const cacheNamespace = 'changelog-github-release';
   function getCacheKey(prev: string, next: string): string {
-    return `${manager}:${depName}:${prev}:${next}`;
+    return `${repository}:${datasource}:${depName}:${prev}:${next}`;
```

Comment:
We still need to think a little on this one. Is perhaps repository enough? manager definitely seems wrong, but I'm uncertain on what's best.

Cc @zharinov

Reply:
Maybe `sourceUrl` is better here than repository+datasource? There's small chance that a self-hosted users has same repository name for public package and private, but with different release notes.